frontend/src/Metamaps/Map/index.js

```diff
@@ -44,6 +44,10 @@ const Map = {
     $('#wrapper').on('contextmenu', function (e) {
       return false
     })
+    
+    $('#wrapper').mousedown(function (e){
+      if(e.button==1)return false;
```

Comment:
You know when you click in the middle mouse, there is a default action in some browsers to star letting you 'pan' around the page... plus the pointer icon changes... this prevents that